CMakeLists.txt

```
@@ -176,13 +176,40 @@ else()
     )
 endif()
 
+set(CPP_ATOMIC_LIBRARY "")
+if(NOT MSVC)
+    # Some platforms require a explicit linkage against libatomic to operate on 64-bit numbers
+    set(TEST_SOURCE "
+#include <cstdint>
```

```suggestion
#include <stdint.h>
```

this is the correct header to include.